src/parser/jobs/dnc/modules/Technicalities.tsx

```diff
@@ -71,28 +72,41 @@ export default class Technicalities extends Module {
 
 	protected init() {
 		this.addHook('applybuff', {to: 'player', abilityId: STATUSES.TECHNICAL_FINISH.id}, this.tryOpenWindow)
+		this.addHook('aoeapplybuff', {by: 'player', abilityId: STATUSES.TECHNICAL_FINISH.id}, this.countTechBuffs)
 		this.addHook('removebuff', {to: 'player', abilityId: WINDOW_STATUSES}, this.tryCloseWindow)
 		this.addHook('cast', {by: 'player'}, this.onCast)
 		this.addHook('complete', this.onComplete)
 	}
 
-	private tryOpenWindow(event: BuffEvent) {
+	private countTechBuffs(event: AoeEvent) {
+		// Get this from tryOpenWindow. If a window wasn't open, we'll open one.
+		// If it was already open (because another Dancer went first), we'll keep using it
+		const lastWindow: TechnicalWindow | undefined = this.tryOpenWindow(event)
```

Comment:
nit; this type annotation is probably unnecessary. looks like `tryOpenWindow` always returns a value now.